pages/index.html

```diff
@@ -4,9 +4,45 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Yangweiyi Li - Home Page</title>
+  <link rel="stylesheet" href="../styles/style.css">
+  <link rel="stylesheet" href="../styles/index.css">
 </head>
+
 <body>
+  <header class="container-a">
+    <h1>Yangweiyi Li</h1>
+  </header>
+
+  <div class="container-b">
```

Comment:
When using semantic HTML I recommend trying to stick with the semantic tags and trying to avoid divs if possible.  This div would work well as a `<main>` tag:
```suggestion
  <main class="container-b">
```